csm-1.5/CASMTRIAGE-6796-csm-pre-install-metal-ipxe/rpm/index.yaml

```diff
@@ -1,3 +1,3 @@
 https://artifactory.algol60.net/artifactory/csm-rpms/hpe/stable/noos/:
   rpms:
-    - metal-ipxe-2.4.8-1.noarch
+    - metal-ipxe-2.4.101-1.noarch
```

Comment:
This mismatches the other rpm index yaml, I'll double check if that's correct or not. Pretty sure this should be 102.

Reply:
This is right. We want to leave this at `2.4.101` and direct users to the newer hotfix.